src/App.cpp

```diff
@@ -228,7 +228,7 @@ int MyApp::OnExit()
         m_luaLog->Close();
         // Check to see if we have lua messages
         if (m_luaMessages) {
-            XWordErrorMessage(NULL, _T("Errors occurred.  See log file: %s"), (const wxChar *)GetLuaLogFilename().c_str());
+            XWordErrorMessage(NULL, _T("Errors occurred.  See log file: %s"), GetLuaLogFilename().c_str().AsChar());

```

Comment:
The safest thing I can think of is to just get rid of the vararg helpers and make the methods in messages.hpp take wxString so the formatting has to be done by the caller (and can be done using wx's safe format methods).